### ISSUE TITLE
Al buscar CUIT en AFIP -> [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify

### ISSUE THREAD

**Issue by lukio** [tryton-ar/account_invoice_ar#62]
Parece ser un problema con *python 2.7.9*
https://bugs.python.org/issue22417
http://stackoverflow.com/questions/27835619/ssl-certificate-verify-failed-error
```
urlopen(afip_url)
*** URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed 
(_ssl.c:581)>
```

**Comment by lukio** [comment]
mas info: https://groups.google.com/d/msg/pyafipws/tb6AIGMsvEY/tJIB2zuKSdAJ

**Comment by lukio** [comment]
Verificar si este fix no rompe instalaciones con python < 2.7.9
